src/quality_result_gui/quality_error_manager.py

```diff
@@ -167,6 +161,17 @@ def _update_info_label(self, status: CheckStatus) -> None:
     def show_dock_widget(self) -> None:
         self._fetcher.start()
         self.dock_widget.show()
+        self.visualizer.initialize_quality_error_layer()
+        self.visualizer.add_new_errors(
+            self.dock_widget.error_tree_view.get_all_quality_errors()
+        )
+        if not self.dock_widget.show_errors_on_map_check_box.isChecked():
+            self.visualizer.hide_errors()
```

Comment:
could this be provided as an argument for the init layer method, so that the layer is added as unchecked if it would be hidden anyway here?